packages/api/src/generated/graphql.ts

```diff
@@ -1751,7 +1752,7 @@ export type QueryArticleArgs = {
 
 
 export type QueryArticleSavingRequestArgs = {
-  id: Scalars['ID'];
+  url: Scalars['String'];
```

Comment:
Could we allow both and make them optional? I'm just trying to think of how we'd have to roll this out. If only the web calls this, i guess we can deploy web at the right time.

Reply:
Yes, so the frontend won't be messed up.

I will make the change